control/bgp.yaml

```diff
@@ -0,0 +1,72 @@
+openapi: 3.0.3
+info:
+  title: BGP Control models
+  description: >-
+    All BGP control schemas
+  version: ^0.0.0
+components:
+  schemas:
+    BGP.Notification:
```

Comment:
I think most of convention followed is "Ping.Ipv4". BGP.Notification: --> Bgp.Notification:

Reply:
updated